### ISSUE TITLE
a11y test doesn't check contrast

### ISSUE THREAD

**Issue by amrocha** [Shopify/polaris-react#1568]
## Issue summary

When we added a pa11y test to the repo, we ran into issues with the contrast rule: ` WCAG2AA.Principle1.Guideline1_4.1_4_3.G18`

It was giving us false positives because of weird floating point logic, where pa11y thinks the contrast ratio is `4.49` instead of `4.5` and fails the rule.

We should figure out why this is happening, and re-enable the rule

cc @alex-page

**Comment by yourpalsonja** [comment]
@amrocha @alex-page can you check and see if this is done and then we can close it? RIPPAGE! 🔥

**Comment by amrocha** [comment]
This has been fixed!
